plugins/headlamp-plugin/lib/index.ts

```diff
@@ -5,4 +5,5 @@ import { Headlamp, Plugin } from '../types/plugin/lib';
 import Registry from '../types/plugin/registry';
 import * as CommonComponents from './CommonComponents';
 
-export { ApiProxy, K8s, CommonComponents, Utils, Plugin, Registry, Headlamp };
+// We export k8s (lowercase) since someone may use it as we do in the Headlamp source code.
```

Comment:
Nice change.

Probably we should (in a separate PR) change this in the examples, and deprecate uppercase K8s?